docs/source/app_developers_guide/creating_sawtooth_network.rst

```diff
@@ -0,0 +1,460 @@
+      --bind component:(component-bind-string} \
+      --endpoint {public-endpoint-string} \
+      --peers {peer-list}
+
+   If you plan to add two or more nodes to the network, you can specify
+   multiple peers in a comma-separated list, as in this example:
+
+     .. code-block:: none
+
+        --peers tcp://172.0.1.3,tcp://172.0.1.4
+
+#. If the XO transaction processor is not running, start it as described in
+   Step 7 of :doc:`docker`.
+
+
+.. _install-second-val-ubuntu-label:
+
+Step 2: Set Up the Second Validator Node
+----------------------------------------
+
+#. Install Sawtooth on the second node, as described in
+   :doc:`ubuntu`.
+
+#. Create your user key:
+
+   .. code-block:: console
+
+      $ sawtooth keygen
+      writing file: /home/yourname/.sawtooth/keys/yourname.priv
+      writing file: /home/yourname/.sawtooth/keys/yourname.pub
+
+#. Create the root key for the validator:
+
+   .. code-block:: console
+
+      $ sudo sawadm keygen
+      writing file: /etc/sawtooth/keys/validator.priv
+      writing file: /etc/sawtooth/keys/validator.pub
+
+If you have additional nodes, repeat this step on those nodes.
+
+Step 3: Start the Second Validator Node
+----------------------------------------
+
+This step starts all the Sawtooth components on the second node. When the second
+validator fully starts, it will peer with the first validator node.
+
+#. Open a new terminal window on the second node, then use the following command
+   to start the validator. Use the actual values for the network and component
+   bind strings, public endpoint string, and peer list, as described in
+   :ref:`prereqs-multi-ubuntu-label`.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth sawtooth-validator \
+      --bind network:{network-bind-string} \
+      --bind component:(component-bind-string} \
+      --endpoint {public-endpoint-string} \
+      --peers {peer-list}
+
+   If you plan to add two or more node to the network, you can specify
+   multiple peers in a comma-separated list, as in this example:
+
+     .. code-block:: none
+
+        --peers tcp://172.0.1.2,tcp://172.0.1.4
+
+
+#. Start the REST API and transaction processors on the second validator node,
+   as shown in this summary of steps 3, 4, and 5 in :doc:`ubuntu`. Open a
+   separate terminal window to start each component.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth sawtooth-rest-api -v
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth settings-tp -v
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth intkey-tp-python -v
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth xo-tp-python -v
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth sawtooth-poet-validator-registry-tp -v
+
+If you have additional nodes in the network, repeat this step on those nodes.
+
+
+Confirm Network Functionality
+=============================
+
+#. To check whether peering has occurred on the network, submit a block query
```

Comment:
typo I think. "block query" should be "peers query"?

Reply:
Yes.  My bad.  I'll correct this in a separate "fix typos" PR very soon now.